_sass/meet-the-program-coordinator.css

```diff
@@ -0,0 +1,64 @@
+* {
+  box-sizing: border-box;
+	font-family: 'Open Sans', sans-serif;
+}
+
+.box-content {
+  margin: 0;
+  padding: 0;
+  background-color: #fff;
+  color: #272b34;
+}
+.box-content {
+  margin-top: 250px;
+  display: block;
+  margin-bottom: 250px;
+  background-color: inherit;
+}
+.br-left > h1 {
+  text-align: center;
+	text-transform: uppercase;
```

Comment:
update indentation

Reply:
GitHub is killing me over indentation, tabs become spaces and are offset.